cql3/functions/user_function.cc

```diff
@@ -54,8 +54,13 @@ bytes_opt user_function::execute(cql_serialization_format sf, const std::vector<
             }
             return lua::run_script(lua::bitcode_view{ctx.bitcode}, values, return_type(), ctx.cfg).get0();
         },
-        [&] (wasm::context& ctx) {
+        [&] (wasm::context& ctx) -> bytes_opt {
             try {
+                for (auto& param : parameters) {
+                    if (!param && !_called_on_null_input) {
+                        return std::nullopt;
+                    }
+                }
```

Comment:
I think it can be moved outside, since it's common to all scripts.

Reply:
ok